Changelog.md

```diff
@@ -39,7 +40,7 @@ Important Bugfixes:
 
 
 Compiler Features:
- * Commandline Interface: Experimental parser error recovery via the ``--error-recovery`` commandline switch.
+ * Commandline Interface: Experimental parser error recovery via the ``--error-recovery`` commandline switch or StandardCompiler `settings.parserErrorRecovery` boolean.
```

Comment:
This is still under 0.5.10!

Reply:
Yes, but this PR does not introduce the feature, it just corrects the old changelog entry.